_data/patcher-landing-page-tabs-dependencies-list.yml

```diff
@@ -0,0 +1,7 @@
+- id: cli
+  title: Use the CLI
+  content: /assets/img/landing-page-patcher/patcher-cli.gif
+
+- id: ui
+  title: Use the UI
+  content: /assets/img/landing-page-patcher/patcher-ui-tf.png
```

Comment:
The image here is a little bit blurry. This seems like a minor issue, but as I discussed [here](https://gruntwork.atlassian.net/browse/WWW-124), in my experience, users absolutely "judge a book by its cover," and blurry images may be seen as unprofessional, uncomfortable, etc (see The Iceberg Secret). 

See [here](https://gruntwork.atlassian.net/browse/WWW-124?focusedCommentId=13843) for ideas on how to make the images not blurry. I think a crisp `.gif` of scrolling through the dependencies would be better. This can be done in a follow-up issue / PR.

Reply:
Oh, also, the UI tab should be the first/default one!

Reply:
Having this image for now would be better?

![Uploading carbon.png…]()